src/resources/SecurityCache/SecurityCache.ts

```diff
@@ -34,6 +35,13 @@ export default class SecurityCache extends Ressource {
         );
     }
 
+    listFilteredEntities(filters: SecurityCacheFilters) {
+        return this.api.post<PageModel<SecurityCacheIdentityModel>>(
```

Comment:
the call to the list uses the `POST` method? Generally we use the `GET` method for a collection of objects for a list.